### ISSUE TITLE
identifying empty cells in microsoft word table

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3228]
**Reported by Shaimaa on 2013-05-12 18:02**
hi all, 
I strongly suggest that NVDA should speak something to inform the user that he / she reaches a blank cell in a table in a word file. NVDA should say something like (blank)
Steps to reproduce:
1. open a table in a word file.
2. make sure to leave empty cells in the table. 
3. navigate the table. 
results
NVDA doesn't say anything to inform the user that this cell doesn't have content, and the user may not know if the cell is really empty, or NVDA just stopped working or lagging. 
Thanks

**Comment by leonardder** [comment]
I can't reproduce in Word 2016. With coordinates enabled, NVDA either reports the row or column number. With coordinates disabled, NVDA reports blank, at least for table nav.
